src/Exception/ProblemDetailsExceptionInterface.php

```diff
@@ -18,21 +18,23 @@
  */
 interface ProblemDetailsExceptionInterface extends JsonSerializable, Throwable
 {
-    public function getStatus() : int;
+    public function getStatus(): int;
 
-    public function getType() : string;
+    public function getType(): string;
 
-    public function getTitle() : string;
+    public function getTitle(): string;
 
-    public function getDetail() : string;
+    public function getDetail(): string;
 
-    public function getAdditionalData() : array;
+    public function getAdditionalData(): array;
 
     /**
      * Serialize the exception to an array of problem details.
      *
      * Likely useful for the JsonSerializable implementation, but also
      * for cases where the XML variant is desired.
      */
-    public function toArray() : array;
+    public function toArray(): array;
+
+    public function jsonSerialize();
```

Comment:
This shouldn't be added: it's inherited from `JsonSerializable`

Reply:
It seems that it was added because PHPUnit's `createMock` does not detect the jsonSerializable otherwise, generating an object that doesn't implement it and throws an error at runtime.

I have fixed it by creating exception mocks with `new class extends Exception implements ProblemDetailsExceptionInterface {...}` instead.